src/components/kpi/kpi.tmpl.html

```diff
@@ -13,6 +14,14 @@
             <strong>{{kpi.layout.text.emphasis}}</strong> - <span class="real-value">({{ getRealValue() }})</span>
           </span>
         </div>
+        <!-- Info KPI -->
+        <div class="kpi-edit">
```

Comment:
As mentioned, remove the container with the kpi-edit class. And move the `.impac-material(@kpi-settings-focus-color, @kpi-settings-error-color);` so it is applied to anything within the `.kpi` class.